client/src/components/profile/components/camper.tsx

```diff
@@ -100,26 +100,33 @@ function Camper({
       <h2 className='text-center username'>@{username}</h2>
       {name && <p className='text-center name'>{name}</p>}
       {location && <p className='text-center location'>{location}</p>}
-      {isDonating && (
-        <p className='text-center supporter'>
-          <FontAwesomeIcon icon={faHeart} /> {t('profile.supporter')}
-        </p>
-      )}
       {about && <p className='bio text-center'>{about}</p>}
       {joinDate && (
         <p className='bio text-center'>
           <FontAwesomeIcon icon={faCalendar} /> {parseDate(joinDate, t)}
         </p>
       )}
-      {yearsTopContributor.filter(Boolean).length > 0 && (
+      {isDonating && (
+        <dl className='badges'>
+          <FontAwesomeIcon icon={faHeart} />
+          <dt className='supporter'>{t('profile.supporter')}</dt>
+          <dd>{t('donate.supporter')}</dd>
+          {isTopContributor && (
+            <>
+              <FontAwesomeIcon icon={faAward} />
+              <dt className='yearsTopContributor'>
+                <Link to={t('links:top-contributors')}>
+                  {t('profile.contributor')}
+                </Link>
+              </dt>
+              <dd>{t('donate.top-contributor')}</dd>
```

Comment:
Why is the top contributor logic within the isDonating logic?

Top contributors are based on contributions to GitHub/Forum/News etc - not financial contributions.

Reply:
the logic will change when the PR is ready for review, I have added it lazily to focus on adding the CSS that will sort the layout

Reply:
Sounds good - thanks for clarifying 💜